spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -633,13 +633,15 @@ def read_failed_lines
         end
       end
 
-      context "when backtrace will generate a security error" do
-        let(:exception) { instance_double(Exception, :backtrace => [ "#{__FILE__}:#{__LINE__}"]) }
-
-        it "is handled gracefully" do
-          expect {
-            with_safe_set_to_level_that_triggers_security_errors { read_failed_lines }
-          }.not_to raise_error
+      if RSpec::Support::RubyFeatures.supports_taint?
+        context "when backtrace will generate a security error" do
```

Comment:
Aye, I wrote it that way first actually. I just saw the conditional form in the place I copied this condition from and decided I should imitate it harder in case there was a reason for it. Happy to use the `:skip` instead though, I prefer it a bit.